windows_impl_linux.go

```diff
@@ -0,0 +1,287 @@
+// Copyright 2017 The Virtual Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// THIS FILE IS AUTOGENERATED BY generator.go
```

Comment:
Please make the comment compatible with the recently adopted [standard](https://github.com/golang/go/issues/13560#issuecomment-277822359).